src/generate_mhq_sample/20_mhq_terr_sample_admin.Rmd

```diff
@@ -0,0 +1,1011 @@
+
+```{r}
+
+check_anb_type <- sample_assessments_anb %>%
+    filter(assessment_status == "in situ assessment") %>%
+    filter(measured) %>%
+    filter(! str_sub(type_observed, 1 , 4) %in% main_type_anb) %>%
+    select(sample_id, type_target, type_observed)
+
+write.table(check_anb_type, "../../output/check_anb_type.txt")
+
+check_anb_type %>% 
+    kable() %>%
+    kable_styling()
+
+```
+
+## Combine ANB and INBO admin table
+
+```{r}
+sample_assessments <- bind_rows(
+    sample_assessments_anb,
+    sample_assessments_inbo
+) %>%
+    mutate(type_target = factor(type_target, levels= levels(types$type)),
+           assessment_status = factor(assessment_status, levels = c("in situ assessment", "orthophoto assessment", "temporary inaccessable", "permanently inaccesable", "no assessment")),
+           sample_source = "habitatmap")
+```
+
+## Create new samples when is_type_target = FALSE and allow_other_type = TRUE
+
+We still have to decide what to do with samples that were measured for another type than the target type and for which the location was changed.
+
+Do we have to document that these are new samples compared to the original set of samples? How?
+
+```{r}
+new_samples <- sample_assessments %>%
+    filter(assessment_status == "in situ assessment" & !is.na(assessment_status)) %>%
+    filter((!is_type_target) & allow_other_type) %>%
+    mutate(sample_id = str_c(location_id, "_", type_observed),
+           type_target = factor(type_observed, levels = levels(types$type)),
+           is_type_target = TRUE,
+           allow_other_type = NA,
+           sample_source = "mhq assessment") %>%
+    select(-year_planned) %>%
+    unique()
+
+```
+
+
+## Add new samples to sample admin table
+
+```{r}
+sample_assessments_update <- bind_rows(
+    sample_assessments,
+    new_samples
+) %>% 
+    arrange(type_target, sac, grts_ranking)
+
+
+```
+
+To do: check grts-ranking of dune samples.
+
+## Handle location changes
+
+How do we handle location changes? Do we create a new sample_id and do we set in the orignal sample_id is_type_target = FALSE? 
+
+## Export
+
+### Overview of all assessed samples
+
+```{r}
+mhq_assessments_summary <- sample_assessments_update %>%
+    filter(!is.na(assessment_status)) %>%
+    filter(assessment_status != "no assessment") %>%
+    filter(sample_source != "mhq_assessment") %>%
+    filter(!is.na(is_type_target)) %>% # sommige inbo measured plots zijn nog niet ingevoerd in inboveg
+    select(sample_id,  grts_ranking, assessment_status, type_target, is_type_target, type_observed, location_change, distance_from_sample, x_orig, y_orig, x_measured, y_measured)
+   
+check <- n_distinct(mhq_assessments_summary$sample_id) == nrow(mhq_assessments_summary)
+check_n <- mhq_assessments_summary %>%
+    group_by(sample_id) %>%
+    mutate(n = n()) %>%
+    ungroup()
+
+write_vc(mhq_assessments_summary, root = "../../n2khab_mhq_data" , file = "mhq_assessments", sorting = c("type_target", "grts_ranking"))
```

Comment:
Added selected preview as a reference for discussion:

```r
> mhq_assessments %>% slice(seq(1,1401,100)) %>% knitr::kable()
```

|sample_id         | grts_ranking|assessment_status  |type_target |is_type_target |type_observed |location_change | distance_from_sample|   x_orig|   y_orig| x_measured| y_measured|
|:-----------------|------------:|:------------------|:-----------|:--------------|:-------------|:---------------|--------------------:|--------:|--------:|----------:|----------:|
|1338805_1330_da   |      1338805|in situ assessment |1330_da     |TRUE           |1330_da       |FALSE           |                  4.9|  61661.6| 221902.1|    61666.0|   221899.9|
|1781173_1330_hpr  |      1781173|in situ assessment |1330_hpr    |FALSE          |NA            |NA              |                   NA|  62301.6| 222030.1|         NA|         NA|
|1036081_2120      |      1036081|in situ assessment |2120        |FALSE          |NA            |NA              |                   NA|  34173.6| 204462.1|         NA|         NA|
|16772913_2190_mp  |     16772913|in situ assessment |2190_mp     |TRUE           |2190_mp       |TRUE            |                  0.0|  32733.6| 203598.1|    32733.6|   203598.1|
|359574_2330_bu    |       359574|in situ assessment |2330_bu     |FALSE          |2310          |FALSE           |                  0.1| 168765.6| 205998.1|   168765.6|   205998.2|
|1762774_2330_bu   |      1762774|in situ assessment |2330_bu     |TRUE           |2330_bu       |FALSE           |                  0.1| 185661.6| 211054.1|   185661.6|   211054.1|
|120530_4030       |       120530|in situ assessment |4030        |TRUE           |4030          |FALSE           |                  0.1| 224765.6| 186926.1|   224765.6|   186926.2|
|2181769_6230      |      2181769|in situ assessment |6230        |FALSE          |NA            |NA              |                   NA| 112029.6| 193550.1|         NA|         NA|
|753630_6230_hmo   |       753630|in situ assessment |6230_hmo    |TRUE           |6230_hmo      |TRUE            |                 31.3| 160157.6| 225582.1|   160126.4|   225579.4|
|912594_6230_hn    |       912594|in situ assessment |6230_hn     |FALSE          |NA            |NA              |                   NA| 224445.6| 194670.1|         NA|         NA|
|7072966_6410_mo   |      7072966|in situ assessment |6410_mo     |TRUE           |6410_mo       |FALSE           |                  0.3| 162429.6| 179534.1|   162429.7|   179534.4|
|4599942_6410_ve   |      4599942|in situ assessment |6410_ve     |TRUE           |6410_ve       |TRUE            |                 50.3| 179069.6| 184846.1|   179068.8|   184795.8|
|1052582_6510_hu   |      1052582|in situ assessment |6510_hu     |TRUE           |6510_hu       |FALSE           |                  0.1| 185885.6| 181038.1|   185885.6|   181038.1|
|10622226_6510_hus |     10622226|in situ assessment |6510_hus    |TRUE           |6510_hus      |FALSE           |                  0.0| 219133.6| 217454.1|   219133.6|   217454.1|
|4322230_7140_meso |      4322230|in situ assessment |7140_meso   |FALSE          |NA            |NA              |                   NA| 218269.6| 198830.1|         NA|         NA|